internal/service/encryptionatrest/resource_test.go

```diff
@@ -107,6 +124,18 @@ func TestAccEncryptionAtRest_basicAzure(t *testing.T) {
 			TenantID:          conversion.StringPtr(os.Getenv("AZURE_TENANT_ID")),
 		}
 
+		azureKeyVaultAttrMap = map[string]string{
+			"enabled":                    "true",
+			"azure_environment":          azureKeyVault.GetAzureEnvironment(),
+			"resource_group_name":        azureKeyVault.GetResourceGroupName(),
+			"key_vault_name":             azureKeyVault.GetKeyVaultName(),
+			"client_id":                  azureKeyVault.GetClientID(),
+			"key_identifier":             azureKeyVault.GetKeyIdentifier(),
+			"subscription_id":            azureKeyVault.GetSubscriptionID(),
+			"tenant_id":                  azureKeyVault.GetTenantID(),
+			"require_private_networking": "false",
+		}
+
 		azureKeyVaultUpdated = admin.AzureKeyVault{
```

Comment:
same for the `admin.AzureKeyVault`. Maybe worth it to have a helper function? Seems like some values are always the same, might help readability of what attributes we are changing